ui/src/pages/domain/manage.js

```diff
@@ -115,6 +117,7 @@ export async function getServerSideProps(context) {
             manageDomains: domains[1],
             pending: domains[2],
             _csrf: domains[3],
+            testfive:domains[4],
```

Comment:
What is testfive?

Reply:
It's just another demo API for testing, I'm going to remove it.